### ISSUE TITLE
Even as an Admin, I am unable to delete posts made by others ON MY NODE. 

### ISSUE THREAD

**Issue by 4nanook** [friendica/friendica#11623]
- [x ] I have searched open and closed issues for duplicates

### Bug Description
Even as an Admin, I am unable to delete posts on my node made by other users on my node.  This is problematic when the material is illegal in my region such as child pornography, advertising prostitution, illegal drugs, or online gambling.  I can delete the offender and eventually their content will be deleted but this can take a long time.

### Steps to Reproduce

1.  Wait for someone to post something illegal.
2. Login as an admin account.
3. Select the delete option for the item in question.

Actual Result:
A pop-up comes up saying I do not have permission.

Expected Result:
Being an admin, I expect it to do as I command and delete the damned post.

### Platform Info

Friendica Version:
Friendica 'Siberian Iris' 2022.05-dev - 1449

Friendica Source: 
Github

PHP version:
PHP 7.4.29 (cli) (built: Apr 28 2022 11:47:05) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.29, Copyright (c), by Zend Technologies

SQL version:
Server version: 10.6.7-MariaDB-2ubuntu1-log Ubuntu 22.04

Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

**Comment by AlfredSK** [comment]
> Select the delete option for the item in question.

Which one? The one in admin panel -> tools -> delete item?

**Comment by 4nanook** [comment]
No, the inline one with the message.  The delete item requires a UUID or
some such and I do not see that at the end of the URL per the command.

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Comment by tobiasd** [comment]
You can use the URL of a posting. The UUID is the last part of the URL of an item, it will be identified automatically by the tool @AlfredSK mentioned.

But I agree that admins should be able to use the item menu to delete an item they spot that does not fit their nodes rules.

**Comment by 4nanook** [comment]
I deleted the user which eventually deleted thier items, but the end of
the URL did not look anything like a UUID.

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Comment by MrPetovan** [comment]
Example URL: `https://friendica.social/display/0a9c236a-6316a52cadb16016-4015d009`

The UUID in this URL is `0a9c236a-6316a52cadb16016-4015d009`.

However, you can also paste `https://friendica.social/display/0a9c236a-6316a52cadb16016-4015d009` in the item deletion field and it will extract the UUID on its own.
